person.rb

```diff
@@ -1,8 +1,9 @@
 require './nameable'
 require './capitalize_decorator'
```

Comment:
- [ ] It would be better if we use `require_relative` instead of the `require` we have currently :100: require_relative is use when we are referencing paths in our current directory